src/js/controller/AppController.js

```diff
@@ -40,32 +46,44 @@ L.DNC.AppController = L.Class.extend({
     _addEventHandlers: function(){
         this.dropzone.fileReader.on( 'fileparsed', this._handleParsedFile.bind( this ) );
         this.forms.on( 'submit', this._handleFormSubmit.bind(this) );
-        this.menu.geo.on( 'click', this._handleGeoClick.bind(this) );
+        this.menu.geo.on( 'clickedOperation', this._handleGeoClick.bind(this) );
```

Comment:
It felt weird listening to `'click'` event on `menu.geo` when the sender was actually an operation button within the geo.menu